receiver/opencensusreceiver/testdata/config.yaml

```diff
@@ -1,19 +1,54 @@
 receivers:
+  # The following entry initializes the default OpenCensus receiver.
+  # The default values are specified here
+  #  https://github.com/open-telemetry/opentelemetry-service/blob/71589202609d7e787244076b631b45e219101867/receiver/opencensusreceiver/factory.go#L47-L56
+  # The full name of this receiver is `opencensus` and can be referenced in pipelines by 'opencensus'.
   opencensus:
+  # The following entry demonstrates configuring the common receiver settings:
+  # - endpoint
+  # - disable-backpressure
+  # For more information on the struct, refer to
+  # https://github:com/open-telemetry/opentelemetry-service/blob/71589202609d7e787244076b631b45e219101867/config/configmodels/configmodels.go#L142-L150
+  # This configuration has a name 'customname' with a full name of 'opencensus/customname'.
+  # To reference this configuration in a pipeline, use the full name `opencensus/customname`.
   opencensus/customname:
+    # The endpoint for this receiver will be: "0.0.0.0:9090".
     endpoint: 0.0.0.0:9090
+    # This receiver doesn't allow for back pressure.
+    disable-backpressure: true
+  # The following entry configures all of the keep alive settings. These settings are use during server initialization.
   opencensus/keepalive:
     keepalive:
       server-parameters:
+        max-connection-idle: 11s
+        max-connection-age: 12s
+        max-connection-age-grace: 13s
         time: 30s
         timeout: 5s
       enforcement-policy:
         min-time: 10s
         permit-without-stream: true
+  # The following demonstrates how to disable back pressure, set maximum limits on stream, message size and connection
+  # idle time.
   opencensus/nobackpressure:
     disable-backpressure: true
     max-recv-msg-size-mib: 32
     max-concurrent-streams: 16
+    keepalive:
```

Comment:
Do we need keepalive here? It distracts from the stated goal for this particular demo receiver.

Reply:
No it isn't needed, but it shows that the settings can be mix and matched.

Reply:
I've added a note explaining that the test config has shown configurations grouped by their structure however all of the settings can be mix and matched like adding the max connection idle setting in this example.

Reply:
👍